src/client.c

```diff
@@ -35,38 +35,39 @@ static const char *hdr2hex(const struct MBuf *data, char *buf, unsigned buflen)
 	return bin2hex(bin, dlen, buf, buflen);
 }
 
-static bool check_client_passwd(PgSocket *client, const char *passwd)
+static bool check_client_passwd(PgSocket *client, const char *provided_passwd)
```

Comment:
More cleanup to easily discern between the user's real password (from Postgres) vs client provided password.